src/jmclient/blockchaininterface.py

```diff
@@ -342,7 +342,7 @@ def __init__(self, jsonRpc: JsonRpc, network: str, wallet_name: str) -> None:
         actualNet = blockchainInfo['chain']
 
         netmap = {'main': 'mainnet', 'test': 'testnet', 'regtest': 'regtest',
-            'signet': 'signet'}
```

Comment:
This is not enough. This way JM will not see a difference between testnet3 and testnet4, will use same orderbook for different networks, etc...

I implemented signet in #794, all these changes are likely needed to add new network.

Reply:
Thanks! I'll look to #794.
Changes was minimal eventually, but I'll try to propose separate network approach.

Reply:
fixed in 996d38f